src/api/common/validate.js

```diff
@@ -24,6 +24,19 @@ function validateAddressAndSecret(obj) {
   }
 }
 
+function validateSecret(secret) {
+  if (!secret) {
+    throw error('Parameter missing: secret');
+  }
+  try {
+    if (!core.Seed.from_json(secret).is_valid()) {
```

Comment:
'passphrase' would be a valid secret here, which I think is discouraged outside of unit tests